src/utils.py

```diff
@@ -16,5 +17,8 @@ def load_bytes_data(addr: PublicKey, conn: Client):
 
 def get_mint_decimals(conn: Client, mint_pub_key: PublicKey) -> int:
     """Get the mint decimals for a token mint"""
+    if mint_pub_key == WRAPPED_SOL_MINT:
+        return 9
```

Comment:
I don't think this makes sense here. Can you conditional to where this is relevant?

Reply:
discussed over Discord, agreed that it's fine for now.